.github/workflows/ci.yaml

```diff
@@ -29,8 +29,21 @@ jobs:
         with:
           go-version: "1.22"
 
+      - name: Install go-junit-report
+        run: go install github.com/jstemmer/go-junit-report/v2@latest
+
+      - run: mkdir junit-xml
+
       - name: Test
-        run: go test ./...
+        run: go test ./... 2>&1 | tee >(go-junit-report -set-exit-code > junit-xml/${{matrix.os}}.xml)
```

Comment:
In the server we use `https://gotest.tools/gotestsum` as a test running, which has a junit output mode. That seems better than parsing `go test` output?

Reply:
Ah, thanks! I'll switch.